apps/src/code-studio/initApp/project.js

```diff
@@ -431,6 +431,10 @@ var projects = module.exports = {
     }
   },
 
+  showHeaderSaveError() {
+    header.showProjectSaveError();
+  },
```

Comment:
Does this method add any value over calling `header.showProjectSaveError()` directly in `showSaveError_` below?

Reply:
It seems we are grouping methods which call into the header, for possible future extraction. however it seems unlikely that will happen any time soon, so I'm ok with inlining.